### PR TITLE
Test changes

### PR DESCRIPTION
**Pull request by Lesenr1**

### CODE REVIEW

File: test/betting/helpers/bootstrap.mligo [hunk 1]
```diff
@@ -35,12 +35,11 @@ let bootstrap () =
 
   (* Boostrapping BETTING contract *)
   let betting_path = "../../../src/contracts/cameligo/betting/main.mligo" in
-  let iBis = Test.run (fun (x : Types.storage) -> x) init_storage in
```
Comment (SuzanneSoy) [inline test/betting/helpers/bootstrap.mligo:38]: Is this `Test.run` not needed anymore?

File: test/betting/helpers/bootstrap.mligo [hunk 2]
```diff
@@ -35,12 +35,11 @@ let bootstrap () =
...
-  let (betting_address, _, _) = Test.originate_from_file betting_path iBis 0mutez in
-  let betting_taddress = (Test.cast_address betting_address : (Betting parameter_of,Types.storage) typed_address) in
-  let betting_contract = Test.to_contract betting_taddress in
+  let orig = Test.originate_from_file betting_path init_storage 0mutez in
+  let betting_contract = Test.to_contract orig.addr in
+  let betting_address = Test.to_address orig.addr in
```
Comment (SuzanneSoy) [inline test/betting/helpers/bootstrap.mligo:40]: Why does orig.addr need a to_address?

File: test/betting/helpers/callback.mligo [hunk 3]
```diff
@@ -34,12 +34,11 @@ let base_storage (bettingAddr : address) : Callback.storage = {
 
 let originate_from_file (initial_storage : Callback.storage) : originated =
     let betting_path           = "../../../src/contracts/cameligo/betting/callback/main.mligo" in
-    let iTres                  = Test.run (fun (x : Callback.storage) -> x) initial_storage in
```
Comment (SuzanneSoy) [inline test/betting/helpers/callback.mligo:37]: Is this `Test.run` not needed anymore?

File: test/oracle/helpers/bootstrap.mligo [hunk 4]
```diff
@@ -50,12 +50,10 @@ let bootstrap_oracle () =
 
     (* Boostrapping Oracle contract *)
     let oracle_path = "../../../src/contracts/cameligo/oracle/main.mligo" in
-    let iBis = Test.run (fun (x : Types.storage) -> x) init_storage in
```
Comment (SuzanneSoy) [inline test/oracle/helpers/bootstrap.mligo:53]: Is this `Test.run` not needed anymore?

File: test/oracle/helpers/callback.mligo [hunk 5]
```diff
@@ -27,12 +27,11 @@ let base_storage : Callback.storage = {
 
 let originate_from_file (initial_storage : Callback.storage) : originated =
     let oracle_path = "../../../src/contracts/cameligo/oracle/callback/main.mligo" in
-    let iTres = Test.run (fun (x : Callback.storage) -> x) initial_storage in
```
Comment (SuzanneSoy) [inline test/oracle/helpers/callback.mligo:30]: Is this `Test.run` not needed anymore?

**Review by SuzanneSoy — APPROVED** [review]
Tentatively approved with questions?

**Review by LigoSuzanneSoy — APPROVED** [review]

**Pull request merged**